src/context/WalletProvider/NativeWallet/components/NativeLoad.tsx

```diff
@@ -153,15 +161,18 @@ export const NativeLoad = () => {
                     />
                   </Box>
                 </Button>
-                <Button
-                  colorScheme='red'
-                  size='xs'
-                  variant='ghost-filled'
-                  mr={4}
+                <IconButton
+                  aria-label={translate('common.rename')}
+                  variant='ghost'
+                  icon={<EditIcon />}
+                  onClick={() => handleRename(wallet)}
+                />
+                <IconButton
+                  aria-label={translate('common.forget')}
+                  variant='ghost'
+                  icon={<DeleteIcon />}
```

Comment:
We'll want to wrap these two icons in a Box to keep them together on the right and prevent them from spacing out when the wallet name is short.